thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -693,23 +692,29 @@ private static HttpData encodeSuccess(ServiceRequestContext ctx,
                                           SerializationFormat serializationFormat,
                                           String methodName, int seqId,
                                           TBase<?, ?> result) {
-        final ByteBufAllocator alloc = ctx.alloc();
-        ByteBuf buf = alloc.ioBuffer(128);
-        final TTransport transport = new TByteBufTransport(buf);
-        final TProtocol outProto = ThriftProtocolFactories.get(serializationFormat).getProtocol(transport);
 
+        final ByteBuf buf = ctx.alloc().directBuffer(128);
```

Comment:
As per our previous conversation, isn't `buffer()` best to leave it up to the allocator what is best for this server?